protocol/src/main/java/org/apache/drill/exec/proto/beans/CoreOperatorType.java

```diff
@@ -58,7 +58,8 @@
     HBASE_SUB_SCAN(33),
     WINDOW(34),
     NESTED_LOOP_JOIN(35),
-    AVRO_SUB_SCAN(36);
+    AVRO_SUB_SCAN(36),
```

Comment:
Is this actually required to make this work? If it is we need to find a way to remove modifying this protobuf definition as a requirement for adding a format plugin, because this removes the ability for format plugins to truely be drop-in pluggable.